index.html

```diff
@@ -965,15 +935,15 @@
               id="q114"
               data-name="About"
               class="q114"
-            ><div key="0">About</div>
+            ><div>About</div>
             </div>
             </div>
             <div class="outer_q115">
             <div
               id="q115"
               data-name="Professional with experience to work as an IT Manager, Support Manager, Help Desk Manager, Project Coordinator Systems, System Analyst, Support Supervisor in the platform Mainframe and Middleware. Working for more than 15 years as a Mainframe and Middleware CIO/CTO/IT Manager."
               class="q115"
-            ><div key="0">Professional with experience to work as an IT Manager, Support Manager, Help Desk Manager, Project Coordinator Systems, System Analyst, Support Supervisor in the platform Mainframe and Middleware. Working for more than 15 years as a Mainframe and Middleware CIO/CTO/IT Manager.</div>
+            ><div>Professional with experience to work as an IT Manager, Support Manager, Help Desk Manager, Project Coordinator Systems, System Analyst, Support Supervisor in the platform Mainframe and Middleware. Working for more than 15 years as a Mainframe and Middleware CIO/CTO/IT Manager.</div>
```

Comment:
- [ ] [OPTIONAL] This is another case that should be a paragraph as already commented in the review thread. Please consider making the necessary changes.